beaconChain/helpers/stateTransitionHelpers.ts

```diff
@@ -24,7 +31,7 @@ type hash32 = Uint8Array;
  * @returns {EpochNumber}
  */
 export function slotToEpoch(slot: SlotNumber): EpochNumber {
-  return Math.floor(slot / EPOCH_LENGTH);
+  return slot.divn(EPOCH_LENGTH);
```

Comment:
is this equivalent to integer division?

Reply:
Yeah, bn.js only supports integer values.

```typescript
new BN(5).divn(2).toNumber() === 2
```